src/lib/REST/Value/ContentTree/LoadSubtreeRequest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+/**
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+declare(strict_types=1);
+
+namespace EzSystems\EzPlatformAdminUi\REST\Value\ContentTree;
+
+use eZ\Publish\Core\REST\Common\Value as RestValue;
+
+class LoadSubtreeRequest extends RestValue
+{
+    /** @var \EzSystems\EzPlatformAdminUi\REST\Value\ContentTree\LoadSubtreeRequestNode[] */
+    public $elements;
```

Comment:
@tischsoic Please not this will affect UI. You have to align your code to use `nodes` instead of `elements` from the `LoadSubtreeRequest` object. I changed this in f2472da.